conf/muchos.props.example.azure

```diff
@@ -0,0 +1,180 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Acknowledged. Till late in the process, we were working off just a single muchos.props.example, but forked it into two files assuming it would simplify the review and test impact. Will bring back the Azure specific changes into single muchos.props.example and thereby have just one file.

Reply:
We have merged these back into a single muchos.props.example file.